src/Silverback.Integration/Messaging/Configuration/SilverbackBuilderExtensions.cs

```diff
@@ -12,6 +13,8 @@ namespace Microsoft.Extensions.DependencyInjection
 {
     public static class SilverbackBuilderExtensions
     {
+        #region WithConnectionTo
```

Comment:
Second class instead of region? :)

Reply:
It makes perfectly sense. Event thought I don't mind regions in this static classes, it would as side effect force to better separate the tests as well.
It's something to be fixed across the entire solution tho. I'm adding this to my "back log".